components/crud-web-apps/jupyter/backend/apps/common/form.py

```diff
@@ -171,6 +187,27 @@ def set_notebook_memory(notebook, body, defaults):
     container = notebook["spec"]["template"]["spec"]["containers"][0]
 
     memory = get_form_value(body, defaults, "memory")
+    memoryLimit = get_form_value(body, defaults, "memoryLimit")
+
+    limit_factor = utils.load_spawner_ui_config()["memory"].get("limitFactor")
+    if not memoryLimit and limit_factor != "none":
+        memoryLimit = str(
+            round((
+                float(memory.replace('Gi', '')) * float(
+                    limit_factor)), 1)) + "Gi"
+        if not memory:
```

Comment:
Same as above, we can remove this check

Reply:
This has been removed in the latest commit.